src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/DefaultProcessor.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.juddata.camel.processor;
+
+import static java.util.Objects.nonNull;
+
+import java.util.List;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.springframework.beans.factory.annotation.Value;
+import uk.gov.hmcts.reform.juddata.camel.exception.RouteFailedException;
+import uk.gov.hmcts.reform.juddata.camel.validator.JsrValidatorInitializer;
+
+@Slf4j
+public abstract class DefaultProcessor<T> implements Processor {
```

Comment:
as this is used only for jsr validation.. can you renamed this to JSRValidationBaseProcessor?

Reply:
ok

Reply:
done